.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
hm this look slike a .vscode specific thing? could you add this to a `.gitignore` and then uncommit this

Reply:
oh no! yea sorry i forgot about this